nservicebus/azure-service-bus/topologies/index.md

```diff
@@ -27,7 +27,7 @@ The `ForwardingTopology` was introduced to take advantage of the broker nature o
 
 Both topologies create a single input queue per endpoint and implement [Publish-Subscribe](/nservicebus/messaging/publish-subscribe/) mechanism. However, there is a significant difference in the way the mechanism is implemented.
 
-Starting from Version 7, no default topology is set by the Azure Service Bus transport. Topology has to be configured using [configuration API](/nservicebus/azure-service-bus/configuration.md).
+Starting from Version 7, no default topology is set by the Azure Service Bus transport. Topology has to be configured using [configuration API](/nservicebus/azure-service-bus/configuration/configuration.md).
```

Comment:
Can we provide some guidance around which topology to select or are the descriptions above enough? I'm thinking something very short to start with:

> Endpoints that are upgraded from Version 6 to Version 7 must use `EndpointOrientedTopology` to ensure that no messages are lost. 

Or something like that. One day we can write a migration guide for moving from Endpoint to Forwarding.